stats.lua

```diff
@@ -392,6 +402,54 @@ local function append_display_sync(s)
 end
 
 
+local function append_filters(s, prop, prefix)
+    local length = 0
+    local filters = {}
+
+    local speed = mp.get_property_osd("speed")
+    if speed ~= "1.00" then
+        length = length + string.len("speed []") + speed:len()
+        filters[#filters+1] = "speed" .. it(" [" .. speed .. "]")
+    end
+
+    if prop == "vf" and mp.get_property_bool("deinterlace") then
```

Comment:
How can it be "good enough" when you have some status that can't even tell you what exactly is happening.
I don't agree with you here. It's supposed to list video filters and "deinterlace" isn't a video filter, yadif is. When something/someone else is doing deinterlacing then that may be so but that doesn't concern a listing of video filters at all?

I'd rather add a deinterlacing indicator somewhere else but it has no place in "video filters" (especially not if something else can perform deinterlacing, then it's outright misinformation).